server/webapp/WEB-INF/rails/webpack/views/pages/pipelines/scm_material_fields.tsx

```diff
@@ -27,7 +27,8 @@ import {
   TfsMaterialAttributes
 } from "models/materials/types";
 import {CheckboxField, PasswordField, TextField} from "views/components/forms/input_fields";
-import {TestConnection} from "views/components/materials/test_connection";
+// import {TestConnection} from "views/components/materials/test_connection";
```

Comment:
I imagine the `Attrs` will allow us to choose between the buttons in future revisions?